Src/zipkin4net/Src/Tracers/Zipkin/JSONSpanSerializer.cs

```diff
@@ -133,6 +147,16 @@ private static void SerializeEndPoint(StreamWriter writer, IPEndPoint endPoint,
 
     delegate void SerializeMethod<T>(StreamWriter writer, T element, IPEndPoint endPoint, string serviceName);
 
+    public class JSONSpanSerializerOption
```

Comment:
I'm not sure it's mandatory here. If you take a look at pkin/src/main/java/zipkin/internal/JsonCodec.java#L95, it's always serialized as a string, not as an int (which was clearly a bug)

Reply:
Oh. That’s great.....